<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Dialog</class>
    <widget class="QDialog" name="Dialog">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>登录</string>
        </property>
        <widget class="QLabel" name="headline">
            <property name="geometry">
                <rect>
                    <x>200</x>
                    <y>110</y>
                    <width>381</width>
                    <height>51</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string>欢迎登录用药助手！</string>
            </property>
            <property name="alignment">
                <set>Qt::AlignCenter</set>
            </property>
        </widget>
        <widget class="QPushButton" name="loginButton">
            <property name="geometry">
                <rect>
                    <x>320</x>
                    <y>400</y>
                    <width>131</width>
                    <height>41</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>微软雅黑</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string>登录</string>
            </property>
        </widget>
        <widget class="QCommandLinkButton" name="toForgetPwd">
            <property name="geometry">
                <rect>
                    <x>400</x>
                    <y>310</y>
                    <width>121</width>
                    <height>48</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Segoe UI</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string>忘记密码？</string>
            </property>
        </widget>
        <widget class="QWidget" name="formLayoutWidget">
            <property name="geometry">
                <rect>
                    <x>250</x>
                    <y>190</y>
                    <width>281</width>
                    <height>102</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>微软雅黑</family>
                </font>
            </property>
            <layout class="QFormLayout" name="formLayout">
                <property name="horizontalSpacing">
                    <number>20</number>
                </property>
                <property name="verticalSpacing">
                    <number>20</number>
                </property>
                <item row="0" column="0">
                    <widget class="QLabel" name="userName">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>用户名</string>
                        </property>
                    </widget>
                </item>
                <item row="0" column="1">
                    <widget class="QLineEdit" name="inputUserName">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>11</pointsize>
                            </font>
                        </property>
                    </widget>
                </item>
                <item row="1" column="0">
                    <widget class="QLabel" name="password">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>10</pointsize>
                            </font>
                        </property>
                        <property name="text">
                            <string>密码</string>
                        </property>
                    </widget>
                </item>
                <item row="1" column="1">
                    <widget class="QLineEdit" name="inputPwd">
                        <property name="font">
                            <font>
                                <family>微软雅黑</family>
                                <pointsize>11</pointsize>
                            </font>
                        </property>
                        <property name="echoMode">
                            <enum>QLineEdit::Password</enum>
                        </property>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QCommandLinkButton" name="toRegister">
            <property name="geometry">
                <rect>
                    <x>250</x>
                    <y>310</y>
                    <width>141</width>
                    <height>48</height>
                </rect>
            </property>
            <property name="font">
                <font>
                    <family>Segoe UI</family>
                    <pointsize>10</pointsize>
                </font>
            </property>
            <property name="text">
                <string>立即注册</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
